projects/addon-doc/directives/scroll-into-view/scroll-into-view.directive.ts

```diff
@@ -19,7 +19,7 @@ export class TuiScrollIntoViewLinkDirective {
                 switchMap(() => this.scroll$),
                 debounceTime(750),
                 filter(shallWe => shallWe && !!tuiGetElementObscures(el)),
-                takeUntil(inject(TuiDestroyService, {self: true})),
+                takeUntilDestroyed(inject(DestroyRef)),
```

Comment:
here and in many other places `DestroyRef` is unnecessary, just `takeUntilDestroyed()`

Reply:
fixed